src/_P097_Esp32Touch.ino

```diff
@@ -5,48 +5,52 @@
 // #################################### Plugin 097: ESP32 Touch ##########################################
 // #######################################################################################################
 
-
-#if defined(SOC_TOUCH_SENSOR_SUPPORTED) && SOC_TOUCH_SENSOR_SUPPORTED
```

Comment:
Why did you remove this check?
This is using the defines made by Espressif in the soc_caps.h file
So it will work on any new platform supporting touch (like the ESP32-P4)

Reply:
Sorry again.. this wasn´t on purpose... i always get confused when resolving conflicts 😳